package.json

```diff
@@ -13,7 +14,7 @@
     "styled-components": "^4.3.2"
   },
   "scripts": {
-    "start": "REACT_APP_API_URL=http://localhost:9000 react-scripts start",
+    "start": "react-scripts start",
```

Comment:
This change needs to be reverted, so we get proper `env` values in development environment.